librz/arch/dwarf_process.c

```diff
@@ -1751,7 +1755,12 @@ RZ_API void rz_analysis_dwarf_preprocess_info(
 		.unit = NULL,
 	};
 	RzBinDwarfCompUnit *unit;
+	RzAnalysisDebugInfo *debug_info = analysis->debug_info;
 	rz_vector_foreach (&dw->info->units, unit) {
+		debug_info->type_by_offset = ht_up_new(NULL, (HtUPFreeValue)rz_type_free);
```

Comment:
It's a bit awkward here.

Reply:
Kind of. But there is no need to have it until the rizin shuts down. 
* it is not used anywhere except `dwarf_process.c`.
* we have copy of all types in `typedb->types`.

so it just takes RAM for nothing. 
The same for `debug_info->base_type_by_offset` and `debug_info->base_types_by_name`.

Basically all the `analysis->debug_info` is like that. Maybe we should do something with it.